src/components/Details.js

```diff
@@ -0,0 +1,77 @@
+import React, { useEffect } from 'react';
```

Comment:
Please add a `Back` button on the Detail page on the Desktop version that should take the user back to the Home page.